src/main/java/com/google/api/codegen/metacode/FieldStructureParser.java

```diff
@@ -143,15 +146,13 @@ private static InitCodeNode parseConfig(
     if (token == '=') {
       fieldNamePos = Math.min(fieldNamePos, scanner.pos() - 1);
 
-      // TODO(pongad): Currently the RHS of equal sign is just arbitrary run of text treated as string, eg
-      //   a.b=https://foo.bar.com/zip/zap
-      // We'll quote the RHS of existing configs, then we can use parseValue here.
-      // For now, just preserve old behavior.
-      // String valueString = parseValue(scanner);
+      // TODO(pongad): Quote the RHS of existing configs, and once that's done use 'parseValue' here
+      // (`String valueString = parseValue(scanner)`). For now we are preserving the previous
+      // behavior,
```

Comment:
nit: line breaks

Reply:
Hmm, google-java-format seems to do a poor job here :(